src/modules/pmc/src/lib/PmcBase.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <string>
+#include <rapidjson/stringbuffer.h>
+
+#include <CommonUtils.h>
+#include <Logging.h>
+#include <Mmi.h>
+
+#define PMC_LOGFILE "/var/log/osconfig_packagemanagerconfiguration.log"
```

Comment:
So originally I left the long name because I consider this as admin facing data. And for admins it may be more obvious to search for the osconfig_packagemanagerconfiguration.log rather than osconfig_pmc.log.

But for the moment I've changed it as you suggested.

Reply:
The local logs are not facing remote admin data. They are local and need to match the local binary name first and foremost. The remote admin can deduct what pmc means from the description in the PnP interface, please remember to add it there